wdl/GatherSampleEvidenceBatch.wdl

```diff
@@ -64,6 +64,12 @@ workflow GatherSampleEvidenceBatch {
     File? baseline_melt_vcf
     File? baseline_scramble_vcf
 
+
+    # Localize reads parameters
+    # set to true on default, skips localize_reads if set to false
+    Boolean run_localize_reads = true
```

Comment:
Similarly would move this input up higher in the inputs list - probably after collect_pesr